pkg/controller/utils/metadata/operator_metadata.go

```diff
@@ -116,12 +115,8 @@ func (omf *OperatorMetadataForwarder) sendMetadata() error {
 	}
 
 	defer resp.Body.Close()
```

Comment:
This is probably redundant since we aren't reading `resp.Body`?

Reply:
I think the user is [expected](https://pkg.go.dev/net/http#Client.Do) to close `resp.Body` in the case of an error